src/main/java/seedu/address/logic/commands/RedoCommand.java

```diff
@@ -0,0 +1,28 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+
+/**
+ * Redos the state of the address book to a previously undone state.
+ */
+public class RedoCommand extends Command {
+
+    public static final String COMMAND_WORD = "redo";
+    public static final String MESSAGE_SUCCESS = "A redo has been performed!";
```

Comment:
I'm not sure how feasible it is but maybe we could tell the user what the action that was redone was, just in case he doesn't know.